### PR TITLE
added-mkdocs-files

### PR DESCRIPTION
**Pull request by moses-kisakye**
I came up with this from issue(s) GH-20 were i thought the changes in the `.md` files, would be better though am looking forward for a feedback!

> Thank you

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
 # Welcome to autobot Documentation
 
+theme:
+    name: mkdocs-material
+    hljs_languages:
+        - yaml
+        - pyhton
```
Comment (HassanBahati) [inline docs/index.md:7]: @moses-kisakye did you mean python??

Comment (HassanBahati) [inline docs/index.md:7]: @moses-kisakye did you mean `python` ?

**Review by HassanBahati — CHANGES_REQUESTED** [review]
I've left a comment @moses-kisakye

File: docs/index.md (hunk 1 above)
Comment (moses-kisakye) [reply docs/index.md:7]: yeah, it was kinda!!?

Comment (HassanBahati) [reply docs/index.md:7]: please rectify it

Comment (moses-kisakye) [reply docs/index.md:7]: Okay, no problem

**Pull request closed without merging**